src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -44,31 +44,45 @@ const collapsedStatuses = _.flow(
 )
 
 const statusCell = workflowStatuses => {
-  const { succeeded, failed, running, submitted } = collapsedStatuses(workflowStatuses)
+  const { succeeded, failed, running, submitted, abort } = collapsedStatuses(workflowStatuses)
 
   return h(TooltipTrigger, {
     side: 'bottom',
     type: 'light',
     content: table({ style: { margin: '0.5rem' } }, [
       tbody({}, [
-        tr({}, [
+        submitted ? tr({}, [
+          td(styles.statusDetailCell, [submittedIcon()]),
+          td(['Submitted']),
+          td(styles.statusDetailCell, [submitted])
+        ]) : undefined,
+        running ? tr({}, [
+          td(styles.statusDetailCell, [runningIcon()]),
+          td(['Running']),
+          td(styles.statusDetailCell, [running])
+        ]): undefined,
+        succeeded ? tr({}, [
           td(styles.statusDetailCell, [successIcon()]),
+          td(['Succeeded']),
+          td(styles.statusDetailCell, [succeeded])
+        ]): undefined,
+        failed ? tr({}, [
           td(styles.statusDetailCell, [failedIcon()]),
-          td(styles.statusDetailCell, [runningIcon()]),
-          td(styles.statusDetailCell, [submittedIcon()])
-        ]),
-        tr({}, [
-          td(styles.statusDetailCell, [succeeded || 0]),
-          td(styles.statusDetailCell, [failed || 0]),
-          td(styles.statusDetailCell, [running || 0]),
-          td(styles.statusDetailCell, [submitted || 0])
-        ])
-      ])
+          td(['Failed']),
+          td(styles.statusDetailCell, [failed])
+        ]): undefined,
+        abort ? tr({}, [
+          td(styles.statusDetailCell, [abortIcon()]),
+          td(['Aborted']),
+          td(styles.statusDetailCell, [abort])
+        ]): undefined
+      ].filter(element => element !== undefined))
```

Comment:
If you're using `&&`, you won't have to filter.

Also, we'd use lodash for this if we were doing it, just to make sure we were returning rather than mutating.